### PR TITLE
 Use module-qualified names for Hyper-V and Storage commands

### PR DESCRIPTION
**Pull request by csandfeld**
This should fix issue #333

### CODE REVIEW

**Comment by iainbrighton** [comment]
@csandfeld Awesome! You might want to make a small change to kick the unit tests again (the v4 tests keep failing intermittently and there must have been a change on the community build agents). Can you add a line to the `Unreleased` section of the `ChangeLog.md` file?

**Comment by csandfeld** [comment]
@iainbrighton sure 👍

**Comment by csandfeld** [comment]
@iainbrighton looks like it is still failing to run the v4 tests. You need me to do anything else?

**Comment by csandfeld** [comment]
@iainbrighton (you may already know this from you own investigations) I looked at the failing v4 tests, and it appear to have randomly failed (more often than not) the tests, since merging PR #319. Since that merge, the build log of every failed v4 test run I have looked at (I haven't looked at through all of them though) show the same IO error from **Register-LabMedia.Tests.ps1**:

```
[15:13:15][Step 1/1]  [-] Error occurred in test script 'C:\TeamCity\BuildAgent\work\2ab121e51b4f3d7e\Tests\Unit\Src\Public\Register-LabMedia.Tests.ps1' 21ms
[15:13:15][Step 1/1]    IOException: The process cannot access the file 'CustomMedia.json' because it is being used by another process.
[15:13:15][Step 1/1]    at Remove-TestDrive, C:\Program Files\WindowsPowerShell\Modules\Pester\Functions\TestDrive.ps1: line 84
```

Followed by an ParameterBinding  exception causing Pester to exit with error code 1 from **Remove-LabConfiguration.Tests.ps1**:

```
[15:13:15][Step 1/1]  [-] Error occurred in test script 'C:\TeamCity\BuildAgent\work\2ab121e51b4f3d7e\Tests\Unit\Src\Public\Remove-LabConfiguration.Tests.ps1' 191ms
[15:13:15][Step 1/1] C:\TeamCity\BuildAgent\temp\buildTmp\powershell5886935862954223392.ps1 : 
[15:13:15][Step 1/1] Cannot validate argument on parameter 'First'. The argument is null, empty, or 
[15:13:15][Step 1/1] an element of the argument collection contains a null value. Supply a 
[15:13:15][Step 1/1] collection that does not contain any null values and then try the command 
[15:13:15][Step 1/1] again.
[15:13:15][Step 1/1]     + CategoryInfo          : InvalidData: (:) [Write-Error], ParameterBinding 
[15:13:15][Step 1/1]    ValidationException
[15:13:15][Step 1/1]     + FullyQualifiedErrorId : ParameterArgumentValidationError,powershell58869 
[15:13:15][Step 1/1]    35862954223392.ps1
[15:13:15][Step 1/1]  
[15:13:15][Step 1/1] Process exited with code 1
[15:13:14][Step 1/1] Process exited with code 1
[15:13:15][Step 1/1] Step Run Tests (PowerShell) failed
```

Bottom line is that it seem to be a problem with a filelock on **CustomMedia.json**. I wonder if this could be fixed by a sleep statement in **Register-LabMedia.Tests.ps1**, or alternatively by checking that the file is no longer locked, before moving on and allowing Pester to remove the TestDrive.

Thoughts?

**Comment by iainbrighton** [comment]
@csandfeld It looks like it can't remove the test drive after the test. You can try adding a delay, pushing and seeing if that helps the v4 tests?

**Comment by csandfeld** [comment]
@iainbrighton So sleeping for 30 seconds solved the problem, while sleeping for 2 did not. We may want to find a better value (would be a bit of trial-and-error, or an alternative way of checking for file lock, in order to avoid having such a long delay. I will try with 15 seconds next.

**Comment by csandfeld** [comment]
Ok ... decreasing to 15 and it failed again.

**Comment by csandfeld** [comment]
Had only changed a typo in 7dc65c3 before pushing the commit (forgot to save after changing the delay value - duh!). Commited e2bcd6b with the sleep delay set to 30 seconds.

**Comment by iainbrighton** [comment]
Sure! That sounds like a sensible plan :)
Confidential Communication: This e-mail is intended only for the use of the addressee named above and may be confidential. If you are not the addressee you must not read it and must not use any information contained in nor copy it or inform any person other than Virtual Engine or the addressee of its existence or contents. If you have received this email in error please delete it. It is the responsibility of the recipient to ensure that this message and its attachments are virus free. Any views or opinions presented are solely those of the author and do not necessarily represent those of the Company, its directors or officers unless otherwise specifically stated.

**Comment by csandfeld** [comment]
Took a while for the tests to complete (build server had problems processing PS v4 jobs all saturday), but they have passed now :-)

**Comment by iainbrighton** [comment]
Thanks @csandfeld 😃. I think we need an issue for the disabled test. I *think* that if we write to the %TEMP% directory instead of the $TestDrive, Pester won't try and clear that up, thus it won't be held open?

**Comment by csandfeld** [comment]
My pleasure @iainbrighton. Will you open an issue for the disabled test, or should I?

**Comment by iainbrighton** [comment]
@csandfeld I'll do it 👍
